GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -751,6 +752,17 @@ private void RevisionInfo_ContentsResized(ContentsResizedEventArgs e)
         private void CommitMessage_ContentsResized(ContentsResizedEventArgs e)
         {
             _commitMessageHeight = e.NewRectangle.Height;
+
+            // at scale factor of 150% there is rendering artifact - the last line is almost lost
+            // at all other scaling factors from 100% to 350% everything is rendered as expected
+            // see https://github.com/gitextensions/gitextensions/issues/6898 for more details
+            //
+            // absent of an obvious way to fix it, hardcode the fix - add an extra space at the bottom
+            if (DpiUtil.ScaleX - 1.5f < float.Epsilon)
+            {
+                _commitMessageHeight += 16;
```

Comment:
Maybe measure actual string height instead?
```
using (var g = Graphics.FromHwnd(rtbxCommitMessage.Handle))
  rowHeight = (int)g.MeasureString("A", rtbxCommitMessage.Font).Height
```

Reply:
I've tried increasing the height of the box by a height of a measured string, as you suggested.
However the effect wasn't particularly eye pleasing - a measured string was 23 pixels.

Another thing, we would consume CPU cycles and memory to measure a string every time we display a commit info.